bin/consul-cli.rb

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env ruby
+# rubocop:disable all
+
+require 'diplomatic_bag'
+require 'optparse'
+
+def syntax
+  printf "USAGE: #{__FILE__} [[action]] [[options]]\n\n" \
+          "#{__FILE__} services list [options]\n" \
+          "#{__FILE__} service status <service-name> [options]\n" \
+          "#{__FILE__} info [options]\n" \
+          "#{__FILE__} nodes list-duplicate-id [options]\n\n" \
+          "Options:\n" \
+          "-c, --consul-server\n" \
+          "-d, --datacenter\n" \
+          "-f, --format\n" \
+          "-t, --token\n"
+  exit 0
+end
+
+def launch_query(arguments, options)
```

Comment:
Split in different files, one per command, use a file for common function (such as output filtering)

Reply:
Done